_posts/2016-12-07-deprecating-nodes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Deprecating Nodes
+author: David Davis
+tags:
+  - release
+  - platform
+  - 3.0
+---
+
+Nodes were introduced in Pulp in 2.1.0 and provided a way to keep multiple Pulp
+servers synchronized. However, Nodes uses separate codepaths from the normal
+sync+publish codepaths which creates at a few problems:
+
+* Nodes is less tested than sync+publish even though it provides a similar
```

Comment:
I know I wrote this sentence but...    s/even though it provides a similar feature set//